ui/web-v2/apps/admin/src/components/Header/index.tsx

```diff
@@ -1,5 +1,7 @@
 import { FC, memo } from 'react';
 
+import { LanguageSelect } from '../LanguageSelect/LanguageSelect';
```

Comment:
For new or changes in the imports, please use the absolute path.

E.g.
```ts
import { LanguageSelect } from '@/components/LanguageSelect';
```

Reply:
got it.